components/layout/hero/hero-textonly.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="hero-textonly__section section--bg-prim-blue">
+  <section class="hero-textonly">
+    <div class="hero-textonly__title">
+      <h1 class="h1--page-titles headline">Blog</h1>
+    </div><!-- .hero-textonly__title -->
+  </section>
+</div>
+
+<div class="hero-textonly__section section--bg-prim-blue">
+  <section class="hero-textonly">
+    <div class="hero-textonly__title">
+      <h1 class="h1--page-titles headline">Client stories</h1>
+    </div><!-- .hero-textonly__title -->
+  </section>
+</div>
```

Comment:
We probably don't need this second example since there's nothing really different about it compared to the first one.